crates/notifier_host/src/item.rs

```diff
@@ -86,8 +86,8 @@ impl Item {
         Ok(menu.upcast())
     }
 
-    pub async fn icon(&self, size: i32) -> Option<gtk::gdk_pixbuf::Pixbuf> {
+    pub async fn icon(&self, size: i32, scale: i32) -> Option<gtk::gdk_pixbuf::Pixbuf> {
```

Comment:
Passing both size and scale through everything in notifier_host seems unnecessary and error prone (e.g. using size directly instead of size * scale, then not finding out until someone with HiDPI notices it looks wrong)? I feel like you can just calculate size * scale in systray.rs then have size throughout here mean size in physical pixels.